### PR TITLE
Add light automation and effect for Ghost Lantern

### PR DESCRIPTION
**Pull request by simonward**

### CODE REVIEW

**Comment by simonward** [comment]
I considered an aura effect to apply the ghost touch effect, but the sentence "If an affected weapon or attack is magical and already has the maximum number of property runes, the wielder can choose one to suppress to gain ghost touch." made that dubious.

File: packs/equipment/ghost-lantern.json [hunk 1]
```diff
@@ -34,7 +34,27 @@
...
         "quantity": 1,
-        "rules": [],
+        "rules": [
+            {
+                "key": "TokenLight",
+                "value": {
+                    "animation": {
+                        "intensity": 4,
+                        "speed": 1,
+                        "type": "torch"
```
Comment (stwlam) [inline packs/equipment/ghost-lantern.json:44]: Generally avoid using the flickering light type

Comment (simonward) [reply packs/equipment/ghost-lantern.json:44]: I based it on bullseye lantern. Is there one you prefer?

Comment (simonward) [reply packs/equipment/ghost-lantern.json]: Updated to torch.

**Pull request merged**